### PR TITLE
Update logic

### PR DESCRIPTION
**Pull request by JWittmeyer**
Best with

gateway: https://github.com/code-kern-ai/refinery-gateway/pull/35
updater: https://github.com/code-kern-ai/refinery-updater/pull/7
frontend: https://github.com/code-kern-ai/refinery-ui/pull/47
refinery: https://github.com/code-kern-ai/refinery/pull/80

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,35 +1,59 @@
...
 
 
 @app.post("/helper")
 def helper() -> int:
-    from submodules.s3 import controller as s3
-    s3.empty_storage(True,True) 
+    pass
+    # util.check_has_newer_version()
+    # loop_functions_between_version("authorizer", "1.0.0", "1.1.0")
+    # test_me()
```
Comment (FelixKirsch) [inline app.py]: can this be removed?

File: app.py [hunk 2]
```diff
@@ -1,35 +1,59 @@
...
 import util
 
 app = FastAPI()
 
 
-class Request(BaseModel):
-    current_version: str
-    target_version: str
+@app.post("/update_to_newest")
+def update_to_newest() -> int:
```
Comment (FelixKirsch) [inline app.py]: From the other api endpoints, I conclude that as return type the type of the actual content of the return value is stated and that the status_code is ignored. In that case, the return value should be string.
```suggestion
def update_to_newest() -> str:
```

**Review by FelixKirsch — APPROVED** [review]

File: app.py (hunk 1 above)
Comment (JWittmeyer) [reply app.py]: removed

File: app.py (hunk 2 above)
Comment (JWittmeyer) [reply app.py]: changed

**Pull request merged**